apps/examples/src/14-persistence/PersistenceExample.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+	Canvas,
+	ContextMenu,
+	TAB_ID,
+	TldrawEditor,
+	TldrawEditorConfig,
+	TldrawUi,
+} from '@tldraw/tldraw'
+import '@tldraw/tldraw/editor.css'
+import '@tldraw/tldraw/ui.css'
+import { throttle } from '@tldraw/utils'
+import { useEffect, useState } from 'react'
+
+const PERSISTENCE_KEY = 'example-3'
+const config = new TldrawEditorConfig()
+const instanceId = TAB_ID
+const store = config.createStore({ instanceId })
```

Comment:
A while back I moved all our examples away from using this global store pattern where we initialise the store out of bounds of the component. It works OK for these sorts of full-screen 1-off demos, but in a real app where you might want to render the component multiple times with different contents it could cause issues, and it won't work at all if you try and render the component in two places at once. I don't want us to promote it to our devs as a best practice.

Instead, the pattern we should encourage (this is how all our internal stuff works) is one where you construct a fresh store within your component from your persisted state (and then maybe use `loadSnapshot` for subsequent react updates to the same document).

That looks something like this:
```ts
const [state, setState] = useState({name: 'loading'})
useEffect(() => {
    const store = config.createStore({initialData: persistedData, instanceId})
    setState({name: 'ready', store })
}, [])

useEffect(() => {
    doPersistenceStuff(state.store)
}, [state])

return <Tldraw store={state.store} />
```